src/main/blackbox/blackbox_fielddefs.h

```diff
@@ -54,6 +59,22 @@ typedef enum FlightLogFieldCondition {
     FLIGHT_LOG_FIELD_CONDITION_LAST = FLIGHT_LOG_FIELD_CONDITION_NEVER
 } FlightLogFieldCondition;
 
+typedef enum SelectField { // no more than 16
+    SELECT_FIELD_PID = 0,
+    SELECT_FIELD_RC_COMMANDS,
+    SELECT_FIELD_SETPOINT,
+    SELECT_FIELD_BATTERY,
+    SELECT_FIELD_MAG,
+    SELECT_FIELD_ALTITUDE,
+    SELECT_FIELD_RSSI,
+    SELECT_FIELD_GYRO,
+    SELECT_FIELD_ACC,
+    SELECT_FIELD_DEBUG,
+    SELECT_FIELD_MOTOR,
+
+    SELECT_FIELD_GPS = 15
+} SelectField;
```

Comment:
Since this is defined in a header and available to other modules that include it, I'd like to see better naming for this enumeration and its elements. Also it doesn't follow our standards with enumerations being named with a trailing `_e`. So maybe `blackboxSelectField_e`.

Also the elements should be named in a similar way. So `BLACKBOX_SELECT_FIELD_PID`, etc.